src/fabric/src/fabric2_db.erl

```diff
@@ -17,11 +17,18 @@
     create/2,
     open/2,
     delete/2,
+    deleted_dbs_info/2,
+    undelete/4,
+    delete_deleted/3,
```

Comment:
This API is getting a bit too specific I think. How about if we pass a `deleted_at` parameter to `delete/2`. So something like `fabric2_db:delete(DbName, [{deleted_at, <<"2020-03-29T12:00:00">>}])`.

For the `fabric2_fdb` API, if the logic gets too funky with combining into a single function then I'd name it `fabric2_fdb:remove_deleted_db(DbName, TimeStamp)`.

Also, in this export group lets move `deleted_dbs_info/2` down below `list_dbs_info/1,2,3`.